microprofile-jwt-auth-tool/src/main/java/org/jboss/eap/qe/microprofile/jwt/auth/tool/JoseHeader.java

```diff
@@ -0,0 +1,91 @@
+package org.jboss.eap.qe.microprofile.jwt.auth.tool;
+
+import javax.json.Json;
+import javax.json.JsonObject;
+
+/**
+ * A class representing a header of JWT
+ */
+public class JoseHeader {
+
+    private final String keyId;
+    private final String type;
+    private final String algorithm;
+
+    private JoseHeader(Builder builder) {
+        this.keyId = builder.keyId;
+        this.type = builder.type;
+        this.algorithm = builder.algorithm;
+    }
+
+    /**
+     * Shortcut for creating valid instance with custom key ID. Useful when you just need a header with valid values.
+     * @param keyId ID of key which will be propagated in this header.
+     * @return new instance of valid header
+     */
+    public static JoseHeader validWithKeyId(final String keyId) {
```

Comment:
Is there any other valid header? Or is this the only one? If there are other possibilities method name should reflect `JWT` and `RS256` option.

Reply:
No, there are no other valid headers.

Reply:
Right. Can you still reflect it in the method name? It is very readable then.

Reply:
In process of removing the `Builder` pattern I also removed this method.